File: C Language/Beta Group Challenges/Group 3/multi_table.c
/**
 * @file multi_table.c
 * @brief print the multiplication table from 1 to 10
 */

/********************/
#include <stdio.h>

/********************/


int main()
{
    int i = 0;
    int j = 0;

    for(i = 1; i < 11; i++) {
        for(j = 1; j < 11; j++) {
            printf("%d\t", i * j);
        }
        printf("\n");
    }
    printf("\n");

    return 0;
}

File: Python/Programming Course/Labs/lab2/multi_table.py
"""
 Program: multi_table.py
 Description: Print Multiplication table with 10 rows and 10 
"""

i = 0
j = 0

for i in range(1, 11):
    for j in range(1, 11):
        multi = i * j
        print("%2s" % multi, end = " ")
    print("")
